.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please use git rm --cache .vscode
Before the git commit

Reply:
Why is clearing the cache needed?